.github/workflows/conftest.yaml

```diff
@@ -9,6 +9,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Validate integration test data via kubeval
+        uses: garethahealy/github-actions/confbatstest@kubeval
```

Comment:
I've merged [#28](https://github.com/redhat-cop/github-actions/pull/28), should we update here to use that?

Reply:
cool. will do that now